src/sub-blocks/LayoutItem/LayoutItem.tsx

```diff
@@ -43,6 +44,13 @@ const LayoutItem = ({
             return null;
         }
         const themedMedia = getThemedValue(media, theme);
+        const mediaWithMicrodata = mergeVideoMicrodata(themedMedia, {
+            name:
+                typeof content.title === 'string'
```

Comment:
Are we can create deserialization on this step for `content`?

Reply:
fixed